engine/model_draft_transactions.go

```diff
@@ -164,6 +165,13 @@ func (m *DraftTransaction) processConfigOutputs(ctx context.Context) error {
 	conditions := map[string]interface{}{
 		xPubIDField: m.XpubID,
 	}
+	// Get the sender's paymail from the metadata, this help when sender has multiple paymails
+	if m.Metadata != nil && m.Metadata["sender"] != nil {
+		parts := strings.Split(m.Metadata["sender"].(string), "@")
```

Comment:
1. Let's check first if conversion to string works, like:
```golang
paymail, ok := m.Metadata["sender"].(string)
if !ok {
 ///dosth
}
//...
```

2. Maybe there is somewhere a function that gets alias from paymail 🤔 (maybe in go-paymail or in utils)

Reply:
Done